react/features/participants-pane/components/native/LobbyParticipantList.js

```diff
@@ -48,17 +48,10 @@ const LobbyParticipantList = () => {
         </View>
     );
 
-    // Regarding the fact that we have 3 sections, we apply
-    // a certain height percentage for every section in order for all to fit
-    // inside the participants pane container
-    const style = participants.length > 1 && styles.lobbyListContent;
-
     return (
         <CollapsibleList
             title = { title }>
-            <ScrollView
-                bounces = { false }
```

Comment:
Why is this no longer necessary?

Reply:
We no longer scroll inside the list because we have a full scroll. Saying this made me realise that we can to View from ScrollView.